test/kb_blast_server_test.py

```diff
@@ -126,24 +249,128 @@ def test_kb_blast_BLASTn_Search_01(self):
+
+        obj_basename = 'BLASTn'
+        obj_out_name = obj_basename+".test_output.FS"
+        obj_out_type = "KBaseCollections.FeatureSet"
+        genomeSet_name = 'test_genomeSet.BLASTn.GenomeSet'
+        expected_hit_cnt = 3
+        
+        genomeInfo_0 = self.getGenomeInfo('GCF_001566335.1_ASM156633v1_genomic', 0)  # E. coli K-12 MG1655
+        genomeInfo_1 = self.getGenomeInfo('GCF_000021385.1_ASM2138v1_genomic', 1)    # D. vulgaris str. 'Miyazaki F
+        genomeInfo_2 = self.getGenomeInfo('GCF_001721825.1_ASM172182v1_genomic', 2)  # Pseudomonas aeruginosa
+        genomeInfo_3 = self.getGenomeInfo('GCF_002950035.1_ASM295003v1_genomic', 3)  # Shigella boydii
+        genomeInfo_4 = self.getGenomeInfo('GCF_000512125.1_ASM51212v1_genomic', 4)  # Escherichia albertii KF1
+        genome_ref_1 = self.get_obj_ref_from_obj_info(genomeInfo_0)
+        genome_ref_2 = self.get_obj_ref_from_obj_info(genomeInfo_1)
+        genome_ref_3 = self.get_obj_ref_from_obj_info(genomeInfo_2)
+        genome_ref_4 = self.get_obj_ref_from_obj_info(genomeInfo_3)
+        genome_ref_5 = self.get_obj_ref_from_obj_info(genomeInfo_4)
+
+        genome_ref_list = [genome_ref_1, genome_ref_2, genome_ref_3, genome_ref_4, genome_ref_5]
+        genome_scinames = ['FOO', 'BAR', 'FOOBAR', 'BLAH', 'BLAHBLAH']
+
+        # create GenomeSet
+        testGS = {
+            'description': 'five genomes',
+            'elements': dict()
+        }
+        for genome_i, genome_ref in enumerate(genome_ref_list): 
+            testGS['elements'][genome_scinames[genome_i]] = { 'ref': genome_ref }
+
+        obj_info = self.getWsClient().save_objects({'workspace': self.getWsName(),       
+                                                    'objects': [
+                                                        {
+                                                            'type':'KBaseSearch.GenomeSet',
+                                                            'data':testGS,
+                                                            'name':genomeSet_name,
+                                                            'meta':{},
+                                                            'provenance':[
+                                                                {
+                                                                    'service':'kb_blast',
+                                                                    'method':'BLASTn_Search'
+                                                                }
+                                                            ]
+                                                        }]
+                                                })[0]
+
+        #pprint(obj_info)
+        target_genomeSet_ref = "/".join([str(obj_info[WORKSPACE_I]),
+                                         str(obj_info[OBJID_I]),
+                                         str(obj_info[VERSION_I])])
```

Comment:
didn't you write a function to make the ws ref earlier in this file?

Reply:
yup.  missed these.  done